### PR TITLE
fix file_list command

### PR DESCRIPTION
**Pull request by darshanime**
broke due to API change

### CODE REVIEW

File: appknox/client.py [hunk 1]
```diff
@@ -144,16 +144,13 @@ def file_get(self, file_id):
         url = 'files/' + str(file_id)
         return self._request(requests.get, url)
 
-    def file_list(self, project_id):
+    def file_list(self, project_id, limit, offset):
```
Comment (dhilipsiva) [inline appknox/client.py]: @darshanime Default params please. Declare a variable and use that in both the default places.

File: appknox/client.py [hunk 2]
```diff
@@ -144,16 +147,14 @@ def file_get(self, file_id):
...
         """
         return list of files for a project
         """
-        files = []
-        project = self.project_get(project_id)
-        file_dicts = project['data']['relationships']['files']['data']
-        for file_dict in file_dicts:
-            files.append(self.file_get(file_dict['id']))
-        return files
+        url = 'files?projectId=%s&limit=%s&offset=%s' % (
```
Comment (ChillarAnand) [inline appknox/client.py]: Use urlencode?

Comment (darshanime) [inline appknox/client.py]: Nice tip. :+1:

**Comment by dhilipsiva** [comment]
@darshanime This did not have `review` label. And I never reviewed. Can you please move this to `review` in waffle board?

**Pull request merged**